### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ratan325**
usr == user
php == PHP
phpunit = PHPUnit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,11 +80,11 @@ $client = new Redmine\Client('http://redmine.example.com', 'username', 'password
 ### Manual
 
 It is also possible to install the library oneself, either locally to
-a project or globally; say, in `/usr/share/php`.
+a project or globally; say, in `/user/share/PHP`.
```
Comment (Art4) [inline README.md:83]: :-1:  this is not a typical path.

File: README.md [hunk 2]
```diff
@@ -80,11 +80,11 @@ $client = new Redmine\Client('http://redmine.example.com', 'username', 'password
...
 
 First, download and extract the library somewhere. For example, the
 following steps extract v1.5.18 of the library into the
-`vendor/php-redmine-api-1.5.18` directory:
+`vendor/PHP-Redmine-API-1.5.18` directory:
```
Comment (Art4) [inline README.md:87]: :-1:

File: README.md [hunk 3]
```diff
@@ -106,10 +106,10 @@ $client = new Redmine\Client('http://redmine.example.com', 'username', 'password
 
 ### Running the test suite
 
-You can run test suite to make sure the library will work properly on your system. Simply run `vendor/bin/phpunit` in the project's directory :
+You can run the test suite to make sure the library will work properly on your system. Simply run `vendor/bin/PHPUnit` in the project's directory :
```
Comment (Art4) [inline README.md:109]: :-1: 
this path would be invalid.

File: README.md [hunk 4]
```diff
@@ -106,10 +106,10 @@ $client = new Redmine\Client('http://redmine.example.com', 'username', 'password
...
 
 ```
-$ vendor/bin/phpunit
+$ vendor/bin/PHPUnit
```
Comment (Art4) [inline README.md:112]: :-1:  same here.

File: README.md [hunk 5]
```diff
@@ -158,9 +158,9 @@ See `example.php` for further examples.
 
 ## User Impersonation
 
-As of Redmine V2.2 you can impersonate user through the REST API :
+As of Redmine V2.2, you can impersonate user through the REST API :
 
-```php
+PHP
```
Comment (Art4) [inline README.md:163]: :-1: 
this line marks a code area with PHP syntax highlighting. This change makes no sense.

File: README.md [hunk 6]
```diff
@@ -178,6 +178,6 @@ $client->setImpersonateUser(null);
 ### Thanks!
 
 * Thanks to [Thomas Spycher](https://github.com/tspycher/) for the 1st version of the class.
-* Thanks to [Thibault Duplessis aka. ornicar](https://github.com/ornicar) for the php-github-api library, great source of inspiration!
+* Thanks to [Thibault Duplessis aka. ornicar](https://github.com/ornicar) for the PHP-GitHub-API library, a great source of inspiration!
```
Comment (Art4) [inline README.md:181]: :-1: 
The name of the library is "php-github-api" , see https://github.com/ornicar/php-github-api

**Review by Art4 — CHANGES_REQUESTED** [review]
I'm not a native english speaker, but some technical changes looks like a search/replace without understanding the meanings.

**Comment by kbsali** [comment]
@Art4 this is the typical useless PR that propagated on github during the hacktober fest!
Theses guys (@ratan325 and others) went on creating shitty pull request to get... a t-shirt! Ridiculous!

**Comment by Art4** [comment]
Oh, I see... :facepalm:

**Pull request closed without merging**